app/src/main/java/ch/epfl/sdp/blindly/main_screen/MainScreen.kt

```diff
@@ -45,4 +47,21 @@ class MainScreen : AppCompatActivity() {
             }
         })
     }
+
+    override fun onBackPressed() {
+        val builder: AlertDialog.Builder = AlertDialog.Builder(this)
+        builder.setTitle("Exit the app.")
+        builder.setMessage("Are You Sure?")
+        builder.setPositiveButton("Yes") { dialog, _ ->
+            dialog.dismiss()
+            this.finishAffinity()
+        }
+        builder.setNegativeButton(
+            "No"
```

Comment:
I think it may be nice to have those strings stored in constants. It could be better to use @string resource, but I don't think it can be used there and that's why I suggest using constants.

Reply:
Sure, always a good practice to use constants, I will address that.